lib/core/macros.nim

```diff
@@ -538,6 +538,7 @@ proc parseStmt*(s: string): NimNode {.noSideEffect, compileTime.} =
 
 proc getAst*(macroOrTemplate: untyped): NimNode {.magic: "ExpandToAst", noSideEffect.}
   ## Obtains the AST nodes returned from a macro or template invocation.
+  ## See also `genasts.genAst`.
```

Comment:
Should be a link.

Reply:
done, but https://github.com/nim-lang/RFCs/issues/125 will make all this obsolete.